modules/tinymce/src/themes/silver/main/ts/ui/window/SilverInlineDialog.ts

```diff
@@ -58,9 +73,33 @@ const renderInlineDialog = <T>(dialogInit: DialogManager.DialogInit<T>, extra: S
   const dialogEvents = SilverDialogEvents.initDialog(
     () => instanceApi,
     {
-      // TODO: Implement block and unblock for inline dialogs
-      onBlock: () => { },
-      onUnblock: () => { },
+      onBlock: (event) => {
+        Blocking.block(dialog, (bs) => ({
```

Comment:
This spec is duplicated with the logic in `SilverDialogCommon`. Can you please move all this logic to once place to remove the duplication?

- [ ] Remove busy spec duplication

Reply:
Deduped